tests/handlers/aws/test_handler.py

```diff
@@ -1764,57 +1769,73 @@ def test_lambda_handler_replay(self) -> None:
             queue_attributes=self._queues_info["source-s3-sqs-queue"], filenames=[first_filename, second_filename]
         )
         event_s3 = _event_from_sqs_message(queue_attributes=self._queues_info["source-s3-sqs-queue"])
+        bucket_arn: str = "arn:aws:s3:::test-bucket"
+        event_time = int(
+            datetime.datetime.strptime("2021-09-08T18:34:25.042Z", "%Y-%m-%dT%H:%M:%S.%fZ").timestamp() * 1000
+        )
+
+        prefix_s3_first = f"{event_time}-{bucket_arn}-{first_filename}"
+        prefix_s3_second = f"{event_time}-{bucket_arn}-{second_filename}"
 
         _event_to_sqs_message(queue_attributes=self._queues_info["source-sqs-queue"], message_body=self._cloudwatch_log)
-        event_sqs = _event_from_sqs_message(queue_attributes=self._queues_info["source-sqs-queue"])
+        timestamp = str(int(datetime.datetime.utcnow().timestamp()))
+        queue_attributes = self._queues_info["source-sqs-queue"]
+        queue_attributes["sent_timestamp"] = timestamp
```

Comment:
I'm curious why this is needed at all: does not `sqs_client.receive_message()` include `messages["Messages"][]["attributes"]["SentTimestamp"]` in the first place?

Reply:
minor: replacing
[`messages = sqs_client.receive_message(QueueUrl=queue_attributes["QueueUrl"], MessageAttributeNames=["All"])`](https://github.com/elastic/elastic-serverless-forwarder/blob/a588ec0995d9a04aab916c81d64693568bb0b7e4/tests/handlers/aws/test_handler.py#L1235) with `messages = sqs_client.receive_message(QueueUrl=queue_attributes["QueueUrl"], AttributeNames=["All"], MessageAttributeNames=["All"])` will prevent the need for `queue_attributes["sent_timestamp"] = timestamp`

We can then have `_event_from_sqs_message` return `event_sqs` and `event_sent_timestamp`, similarly to what we do with `_event_from_cloudwatch_logs`

Reply:
You are totally right. I did not think about that :).